LaunchAtLogin/copy-helper.sh

```diff
@@ -1,7 +1,15 @@
 #!/bin/bash
 
-origin_helper_path="$BUILT_PRODUCTS_DIR/$FRAMEWORKS_FOLDER_PATH/LaunchAtLogin.framework/Resources/LaunchAtLoginHelper.app"
-helper_dir="$BUILT_PRODUCTS_DIR/$CONTENTS_FOLDER_PATH/Library/LoginItems"
+if [ "${DEPLOYMENT_LOCATION}" = "YES" ] ; then
+    WHERE="${DSTROOT}"
```

Comment:
`WHERE` is too generic. Make a descriptive variable name. Should also be lowercase.